### ISSUE TITLE
recipe for target 'drivers' failed

### ISSUE THREAD

**Issue by MartinBagge** [madisongh/meta-tegra#127]
Hi,

I'm trying to build an image for a Jetson Xavier.
At some point in the build process i get this error:
build/tmp-glibc/work-shared/jetson-xavier/kernel-source/Makefile:1102: recipe for target 'drivers' failed
Do I need to download something to get the drivers?
Or set some variable?
I have already set the NVIDIA-DEVNET-MIRROR and other variables specified.

**Comment by madisongh** [comment]
Which gcc version are you using? I'll bet there are some issues compiling NVIDIA's kernel with gcc 8, and I've been using gcc 7 so far in my testing.

**Comment by MartinBagge** [comment]
I just tried to create docker a container for the building and made sure that the environment had gcc 7.4 only and output of gcc --version is also 7.4
I tried specifying the GCCVERSION in local.conf but as I think you mention on the wiki the package containing gcc 7 can't be found.

I am still getitng the same drivers error. I have attached the log file for further information on it if it helps.

[log.do_compile.10447.txt](https://github.com/madisongh/meta-tegra/files/3141792/log.do_compile.10447.txt)

Thanks for spending the time to help out :)
